src/main/java/io/devlabs/keytree/domains/company/domain/CompanyAccount.java

```diff
@@ -0,0 +1,40 @@
+package io.devlabs.keytree.domains.company.domain;
+
+import io.devlabs.keytree.commons.jpa.BaseTimeEntity;
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotNull;
+import lombok.*;
+import org.hibernate.annotations.Comment;
+
+@Entity
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@AllArgsConstructor
+@Getter
+@Builder
+public class CompanyAccount extends BaseTimeEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Comment("회사 계정 고유 아이디")
+    private Long id;
+
+    @Column
+    @NotNull
+    @Comment("계정 이름")
+    private String name;
+
+    @Column
+    @NotNull
+    @Comment("로그인 아이디")
+    private String loginId;
+
+    @Column
+    @NotNull
+    @Comment("로그인 비밀번호")
+    private String password;
+
+    @Column
+    @NotNull
+    @Comment("권한")
+    private Authority authority;
+}
```

Comment:
```suggestion
// 클래스명 -> Admin 또는 AdminUser
public class Admin extends BaseTimeEntity {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Comment("관리자 아이디")
    private Long id;

    @Column
    @NotNull
    @Comment("관리자명)
    private String name;

    @Column
    @NotNull
    @Comment("이메일")
    private String email;

    @Column
    @NotNull
    @Comment("비밀번호")
    private String password;

    // 변경된 enum에 대한 로직 추가
}
```
- enum을 엔티티 필드에서 컬럼으로 매핑하는 방식을 찾아보고 적용해주세요.

Reply:
엔티티에서 enum 매핑할 때 @Enumerated 어노테이션 적용하여 수정했습니다.